rust/relay/src/server.rs

```diff
@@ -179,6 +174,11 @@ const UDP_TRANSPORT: u8 = 17;
 /// See <https://www.rfc-editor.org/rfc/rfc8656#name-channels-2>.
 const CHANNEL_BINDING_DURATION: Duration = Duration::from_secs(600);
 
+/// The timeout before a channel be rebound.
```

Comment:
Is that "is rebound" or "can be rebound"?

I'm guessing it's meant to prevent a new client from getting messages mixed up with an old client that's just freed up a channel?

Reply:
> I'm guessing it's meant to prevent a new client from getting messages mixed up with an old client that's just freed up a channel?

Correct. Data relayed on channel bindings is unauthenticated for efficiency reasons which is why they must be refreshed every 5 minutes. Because it is unauthenticated, the spec says you must wait for 5 minutes after it has expired before you give the same channel number to somebody else.

Reply:
Okay, so "can be rebound"